NEW_README.md

```diff
@@ -0,0 +1,9 @@
+# 🏃🏽‍♀️🏃🏾‍♂️🏃🏻‍♀️💨 **_⸻ RUN_**
+
+![GitHub go.mod Go version](https://img.shields.io/github/go-mod/go-version/amonks/run?logo=go&logoColor=white&label=%20&labelColor=gray&color=00ADD8)
+[![Godoc](https://img.shields.io/badge/go-docs-blue?logo=go&logoColor=white&label=%20&labelColor=gray&color=blue)](https://amonks.github.io/run)
+[![Go Report Card](https://goreportcard.com/badge/github.com/amonks/run)](https://goreportcard.com/report/github.com/amonks/run)
+
+Run is a task runner that simplifies executing and managing tasks defined in `tasks.toml` files, with a focus on development workflows. It features an interactive UI for long-lived processes and a non-interactive output mode for short-lived tasks, making it versatile for a wide range of development tasks.
```

Comment:
I agree strongly that the existing copy is bad.

I like the go.mod and godoc badges. I'm hesitant to include the report card badge without also including its checks in CI — it would be easy to start failing and not notice.

I like including "task runner" and "development workflows" and "wide range of development tasks"

I like removing "mouse support" — it's a weird feature to call out explicitly, vs just saying the ui is nice

I have mixed feelings about "focus on development workflows" — I use run in prod too, although I'm not sure I'd recommend this to others :)

I take slight aesthetic issue with "simplifies executing tasks defined in tasks.toml files" — that isn't something most projects do, so it obvious why simplifying it is useful.

I think ideal copy would also include something about dependency sequencing, altho I'm not sure the best way to do this (via comparison with other tools? via a very short description? can it even be captured in a very short description?) — either way, the copy doesn't have anything like this either before _or_ after this diff, so it isn't salient to the review.

***

I think this change is an improvement 👍

Reply:
I agree with all of this, and only included "focus on development workflow" because that's the only setting I personally use run in _for now_  :)